### DISCUSSION TITLE
strange behaviour Victron 100/20 with variable cloudiness

### DISCUSSION THREAD

**Discussion by peff74** [hoylabs/OpenDTU-OnBattery#513] [Q&A] [answered]
The topic actually has nothing to do with OpenDTU.
But because a lot of people here use Victrons, I'd like to ask a question.

It seems to be a well-known phenomenon that the Victron 100/20s go down once every 10 minutes and the voltage drops.
![2023-10-21 19_16_39-Program Manager](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/6355ef02-aaaf-4edc-9093-512237c0cfff)

If this happens at a moment when the cloud cover is changing rapidly (April weather), the Victron hangs at 3-4W charging power.
Often it manages to recover at the next 10min interval. But sometimes it takes much longer... which is really annoying!
What leads to success much faster is to deactivate the charger in the APP and activate it again.
![2023-10-21 19_23_38-Screenshot_20231021_145403 jpg ‎- Windows-Fotoanzeige](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/dffe565f-a35a-4916-ba61-843cd4cb46fe)

Do you have similar experiences or even a solution for this?

**Comment by Manos1966** [comment]
Korrekt! Jede 10Min... (Victron 150/35)
![Victron 10min reset](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/9db36164-d394-4d95-8712-71e2ecfa5852)

**Reply by Napizaki** [reply]
The behavior every 10 min is normal - that is the MPPT looking for the new maximum power point.

The behavior from your lower diagram is not normal - I also have this on my 100/20 and am almost desperate. I have also already contacted Victron support - but there no one really took me seriously and they said that I should buy a Cerbo GX to it (bullshit).
My charge controller is also not defective - I have already replaced it and the new one shows exactly the same behavior.
I have tried forever around and could not solve the problem directly.
Then I made a workaround with Homeassistant:
If the PV voltage (in my case) greater than 73V (open circuit voltage) and the MPPT is in "bulk" and produces a power of less than 5W, then I switch a relay that disconnects the PV input from the Victron for 15 sec. and then reconnects - then it immediately continues normally.

Too bad that Victron does not take you seriously. I assume that it is a firmware bug - but for 6 months there was no update (have v161 on it).

With a friend these problems do not occur with the 150/35.

**Reply by Manos1966** [reply]
@peff74  & @Napizaki Thank you for this very interesting Information. 
I will add it to the Wiki, in case there is more people experiencing this.

The 150/35 seems to behave normal.
unfortunaly I do not have a 100/20 to test
![150-35 normal](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/51e15e86-2167-4f08-ac7b-41f50c867af7)

**Comment by peff74** [comment]
yes I am also 100% sure that this is a bug.
I have also tried to contact Victron about this issue, but support is only through the dealers and they just replace the units.

But to change to a 150/35 because of this?

I have already thought about solving the problem via a serial port of the Victron. You can use it to activate or deactivate the "charger".

Maybe it could be integrated into OpenDTU OnBattery^^

**Reply by greymda** [reply]
@peff74  sorry for hijacking the thread, but quick question, i saw you use a JK BMS BD4A17S4P - can you confirm it's working as expected in opendtu-onbattery please? thanks!

**Reply by peff74** [reply]
Oh, I don't use the JK BMS add-on in opendtu-onbattery.
Because it is not possible to change the settings via serial interface.
I use [esphome-jk-bms](https://github.com/syssi/esphome-jk-bms) as standalone version.

**Reply by greymda** [reply]
@peff74  thanks for getting back to me. it’s the only model i found locally, other are to be ordered from china and have no idea if this will work. wondering if its supported by that project it should be covered by this project as well, but that’s only an assumption..

**Comment by schlimmchen** [comment]
> yes I am also 100% sure that this is a bug.
> Maybe it could be integrated into OpenDTU OnBattery^^

Are you asking to fix something in OpenDTU-OnBattery that you declare to be a bug in an outside commercial product?

My MPPT 150|35 does it as well:

![Screenshot from 2023-10-24 20-23-14](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/3578416/a2ee81dc-b18a-48c9-a594-50bfa6cb30c2)

And I don't see any reason whatsoever to be nervous about it.

I agree that it is infuriating that Victron does not explain what happens there, but frankly, why should we care about these dips? There is probably a good reason.

> With a friend these problems do not occur with the 150/35.

Well, then it's not model-specific, but firmware-specific or specific to some other property, since mine is a 150|35.

**Reply by Manos1966** [reply]
@schlimmchen I think the issue they raised is that, while our 150/35 checks and re-starts producing very quickly, the 100/20 **takes a very long time to re-start with production**, sometimes up to 20min as the red circles in the graph above show.

**Reply by schlimmchen** [reply]
Whoops, I missed that. Thanks for pointing this out to me! Of course, a multi-minute dropout is unacceptable.

Still, I am not very happy about the idea of trying to address this in OpenDTU-OnBattery. It depends on #479 for sure. But even then... An opt-in feature that sends charger disable and enable commands *when* exactly? If the battery output was less than how many Watts for how many minutes? What about at night? Meh...

**Reply by peff74** [reply]
It was rather jokingly spoken about the integration into OpenDTU-OnBattery ;-)
I really hope for a fix from Victron.

But if someone wanted to integrate it, the condition finding would be quite simple.
Deactivate the charger for 10sec. 
If more than 2min the following conditions are met:
- PV voltage above 75V 
- MPPT is in bulk mode
- Victron Power below 5W

Battery voltage is not of interest here.
PV does not supply voltage at night ;-)

**Reply by Manos1966** [reply]
honestly @peff74 many thanks for mentioning this particular performance of the 100/20

If this happens with all Victron 100/20, this is definitively a serious argument **against** using 100/20 during the winter period (where there is only a few hours of light to charge the battery, about half of what is during summer [hrs charging](https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Battery%E2%80%90Tipps#how-much-power-do-i-need-to-charge-my-battery)  ).

**Reply by Manos1966** [reply]
Relax @schlimmchen nobody expects OpenDTUonBattery can cure everything (although a post comes to mind... 🙈🙈🙈🙈)
On the other side, if it could cure cancer... 🤩 😂😂😂😂

**Comment by peff74** [comment, marked as answer]
I think my question has been answered, I am not alone and the problem is clearly identified.
Hopefully Victron will release a bug fix some day.
I've been wondering for days what the problem could be...let's hope this thread helps others to easily identify the problem.
And I now know that the 150/35 does not have this problem, and I have an alternative :-)

**Reply by peff74** [reply]
the 150/35  seems to be affected as well...
![2023-10-25 12_58_33-Window](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/8be0d7b4-e4a1-47a6-8e3e-0ea3d4157af2)

**Comment by Napizaki** [comment]
Here is another diagram of what exactly happens:

![WhatsApp Bild 2023-10-25 um 07 29 51_7d52b9fb](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/126354867/3a804244-ff7b-4ff8-9335-0522e31d79b2)

Sometimes it even takes hours for the 100/20 to start producing normally again on its own.

**Reply by peff74** [reply]
Jep, that's exakt the issue.
That really sucks.

Maybe we should try to annoy Victron again.

**Comment by Manos1966** [comment]
I am not sure I can confirm this happens with the 150/35

The only moment I could identify such a drop is when the Victron 150/35 identified that the Pylontech Battery reached its max charge level (which is normal).

![Verhalten 150-35](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/1b323e54-5c8d-4ba9-a545-ced1e5115ad5)

**Reply by peff74** [reply]
wait for April Wetter ;-)

**Reply by Manos1966** [reply]
you make me regret having purged the database just 2 weeks ago... :-(

**Reply by peff74** [reply]
in Herbst it is often April Wetter ;-)
Just take a look in your data when the  cloud change quickly to sunshine.

**Reply by Manos1966** [reply]
Uebrigens, ist es empfehlenswert, die Fehler-Meldungen des Victron auch zu betrachten.
Ich hätte auch gedacht, dass mein 150/35 am 23 Oktober ein Fehl-Verhalten ausweißt (auf Basis der Grafiken).
....haette ich nicht die Fehler-Meldung in meiner Datenbank gefunden ;-)
![Verhalten 150-35 all days](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/c264946e-d219-44ad-b92a-eb774903ebc2)

Ich habe leider die komplette Datenbank am 19en geloescht um Platz zu machen.
So lange ich aber mich erinnern kann, der 150/35 hatte nie das Fehlverhalten das die 100/20 zeigt, bzw. ich konnte immer den Grund fuer das Fehlverhalten finden (wie z.B. am 23en Oktober).

**Reply by peff74** [reply]
Klar, habe ich geschaut ob der Victron in der Situation Fehler zeigt.
Habe es ja live erlebt und stand quasi daneben.
Aber nix, alles grün. Und Ladestatus war Bulk.

@Manos1966 
Überspannung an Laderegler is net good.
You have to mod you system, immediately.

**Comment by peff74** [comment]
today wieder April weather :-(

![2023-11-16 10_53_39-mRemoteNG - confCons xml - 127 0 0 1](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/a2fa201b-5568-4346-8812-fa893b418a9f)

**Reply by Manos1966** [reply]
:-(

**Comment by ottelo9** [comment]
Here is the same problem (unsolved):
https://community.victronenergy.com/questions/158791/de-smartsolar-10020-tracker-hangt-bei-wechselnder.html

**Comment by Manos1966** [comment]
Dear @peff74 and @Napizaki 
I want to thank you for sharing this important information at the forum.
Thanks to your explanations I refrained from buying the 100/20 I initially planned for.
I acquired some 150/35 instead (thanks to @madmartin I even acquired them at the price I expected to pay for the 100/20) 

I am sure you spared me from a lot of headaches! ❤️ 

...although I am not sure how many (spared) because I read 🤦‍♂️ 

> [Thomas](https://community.victronenergy.com/users/35169/electronictom.html) answered · Nov 28 2022 at 3:17 PM
Das Fehlverhalten kann ich bestätigen, bei meinem BlueSolar **MPPT 150/35 (FW:1.59)** tritt es ebenfalls auf.

**Reply by ottelo9** [reply]
It's a real shame that Victron isn't doing anything about it. It may also be an unfixable hardware problem.

**Comment by ottelo9** [comment]
Ich habe das Problem letzte Woche dem Victron Support gemeldet. Mit allen Details, Bildern und das ich den MPPT bereits gegen einen neuen baugleichen getauscht hatte. Ich habe auch diverse Foreneinträge mit dem Problem angegeben.

Hier ist die Antwort:
```
Victron energy hat uns ihre Anfrage weitergeleitet.
Eines vorweg : Uns sind hier keine anderen Reklamationen diesbezüglich bei diesem MPPT von anderen Anwendern bekannt. Auch liegt uns seitens des Herstellers keine Info vor, das es hier zu sporadischen Ausfällen kommen kann. Allgemein schalten die MPPT immer dann die Leistung weg, wenn ein Temperatur Problem vorliegt oder eine zu hohe Eingangsspannung anliegt !
Da sie ja bereits verschiedene andere Dinge geprüft haben und ausschließen können, müsste die Installation darauf hin überprüft werden. Ansonsten können sie uns den Solarregler gerne zur Überprüfung einsenden. Sollte ein Defekt vorliegen, wird das Gerät im Rahmen der Gewährleistung getauscht.

FraRon electronic GmbH
Serviceteam
service@fraron.de
Martin Englert
```

Also bitte alle, die das Problem haben beim Victron Support UND direkt beim FraRon Support melden. Sonst werden die das Problem wohl nie lösen.

**Reply by peff74** [reply]
Warum beim FraRon Support?

**Reply by oldman4U** [reply]
Ist es eigentlich bekannt ob es das Problem "nur" bei bestimmten Spannungen gibt? Ein Panel mit ca. 40V an einer 24V LiFePO4 hatte ich das Problem auch. Werde es demnächst mit 2 Paneelen an einer 48V Batterie versuchen.

**Reply by peff74** [reply]
nein, es tritt in jeder Konstellation auf

**Reply by oldman4U** [reply]
OK. Danke

**Reply by Napizaki** [reply]
Ich kann bestätigen, dass es auch bei 2 Panel an 48V auftritt. Mein 150/35 hat bei mir nie ein Problem gemacht - mein 100/20 andauernd - egal was ich ändere. Der wird bei mir wohl bald ausgemustert. Hatte echt große Hoffnung in das Update - leider bestätigt sich bei mir ebenfalls, dass das Problem noch da ist. Ich hatte schon vor 6 Monaten Kontakt mit Victron - leider konnte (wollte) mir da niemand helfen - anstatt den Fehler bei Ihnen selbst zu suchen, wurde Batterie, Panel, Verkabelung, ... als Fehlerquelle angesehen. Komisch nur, dass es funktioniert, wenn man den 150/35 einbaut. Zudem ist der 100/20 mit meinem Setup noch weit von seinen Grenzen entfernt.

**Reply by ottelo9** [reply]
Bitte alle (nochmal) ein Support Ticket bei Victron erstellen. Vielleicht hilft es ja. 

@peff74 
#Warum beim FraRon Support?
Ich dachte bis eben, dass wäre der deutsche Support ^^. Aber da habe ich den Laderegler her und deshalb hat Victron die Anfrage wohl dahingeschickt. Also nur beim Victron Support melden und ein Ticket eröffnen. Glaube sonst passiert da nie was.

**Comment by knopers1** [comment]
für den Victron 100/20 gibt es ein Firmware-Update! Ich bin gespannt, ob die Aussetzer damit behoben sind!

**Reply by peff74** [reply]
Ja es scheint nun die 1.63 verteilt zu werden, die im Victron Forum beschrieben wurde.

**Comment by knopers1** [comment]
Okay, die Aussetzer sind immer noch da ! Neue Firmware hat leider das Problem nicht behoben.

**Reply by Snusme83** [reply]
Bei mir sind die Aussetzer auch noch da. Neue Firmware hat auch bei mir nix bewirkt.

**Reply by peff74** [reply]
Wenn dem so ist, gleich Themen im Victron Forum eröffnen.
Im englischsprachigen Bereich, alles andere ist zwecklos.

**Comment by peff74** [comment]
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/fff91d35-3f6c-4a80-aebf-5f03fceb0621)
Also gefühlt ist es besser geworden.
Heute viel Sonne/Wolken und es kam nur einmal und nur für 10min zu dem Problem.

**Reply by knopers1** [reply]
tja, heute hat es mich auch nur einmal erwischt. Vor drei Tagen waren es gleich fünf mal a 10min.

**Comment by knopers1** [comment]
Moin,
ich habe beim Victron dazu einen Ticket eröffnet und habe das Problem deutlich geschildert. 
Obwohl ich auf diverse Foreneinträge verwiesen habe, behauptete der Service Mitarbeiter das Problen nicht zu kennen. 
Die Aussage dazu war: Es gibt nur ein relevantes Victron Forum: die Victron Community...

Und dort ist der Fehler leider nicht thematisiert worden! ??? Ich darf jetzt mein Victron zur Überprüffung einsenden - und bekomme ihn wahrscheinlich als okay wieder...

Wenn Ihr Druck auf Vicreon ausüben wollt, müsser wir gemeinsam das Problem bei Victron Community darstellen, sonst kann jeder von uns einzeln das Teil für nichts einsenden!

**Reply by oldman4U** [reply]
Ich mache da gerne mit, allerdings kommt meiner erst in 2-3 Wochen wieder zum Einsatz.

**Reply by ottelo9** [reply]
@knopers1 
Danke das du dich dort auch gemeldet hast. Du kannst gerne auf deren eigenes Forum verweisen:
https://community.victronenergy.com/questions/158791/de-smartsolar-10020-tracker-hangt-bei-wechselnder.html

**Reply by knopers1** [reply]
super! Danke Dir!

**Comment by greymda** [comment]
i don't know if was already posted, there's a beta firmware that is said to fix the issue (use at own risk, obviously):
 https://www.dropbox.com/scl/fi/mzdszmy8tzum1wwq1ifup/Solar-Charge-Controller-MPPT-100-20-SMART-48V-vC1.64_A060.dup?rlkey=zbna362a7qrzct637qiit4qrz&dl=0

**Reply by peff74** [reply]
interesting to read that the fix was not even included in the official 1.63...Victron really does not cover itself with glory in this topic
But how do you get this 1.64 onto the loader controller?

**Comment by oldman4U** [comment]
@greymda

This would be great, thank you!

Do you know, if there is a changelog or something similar available, or just the .dup?

**Comment by knopers1** [comment]
bin gespannt ob die Firmware das Problem behoben hat. Ich berichte Mal in paar Tagen....

![100_20](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/128924216/8aaa90f5-a0b9-4a57-aa84-3f1f914e6333)

**Reply by ottelo9** [reply]
Das Problem ist mit der neuen 1.64Beta gelöst. Mehr Infos hier: https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Warning-about-Victron-MPPT-100-20
